example.dev-config.yml

```diff
@@ -37,5 +37,5 @@ pulp_source_dir: "/home/vagrant/devel/pulpcore"
 
 pulp_settings:
   secret_key: "unsafe_default"
-  content_origin: "http://{{ ansible_fqdn }}"
+  content_origin: "{{ pulp_webserver_disable_https | default(false) | ternary('http', 'https') }}://{{ ansible_fqdn }}"
```

Comment:
I just realized, we should update these also:

./playbooks/example-use/group_vars/all

./playbooks/example-source/group_vars/all

The docs.

I feel this should be a separate, documented change though. And with a comment (since they are user visible).